src/connection_string.ts

```diff
@@ -1163,14 +1119,6 @@ export const OPTIONS = {
       return fs.readFileSync(String(value), { encoding: 'ascii' });
     }
   },
-  tlsCertificateFile: {
```

Comment:
On line 350 there's handling for this option:
```ts
  if (
    allProvidedOptions.has('tlsCertificateKeyFile') &&
    !allProvidedOptions.has('tlsCertificateFile')
  ) {
    allProvidedOptions.set('tlsCertificateFile', allProvidedOptions.get('tlsCertificateKeyFile'));
  }
```
We should be able to remove this

Reply:
Removed.